ui/nuxeo-document-preview.js

```diff
@@ -279,7 +279,8 @@ import './viewers/nuxeo-video-viewer.js';
         this.document &&
         this.document.properties &&
         this.document.properties['vid:transcodedVideos'] &&
-        this.xpath === 'file:content'
+        this.xpath === 'file:content' &&
+        this.document.properties['file:content']['mime-type'].match(/video.*/)
```

Comment:
Not sure if this helps. If I understood correctly the ticket, we can have a main blob that is a video and then an audio blob on the `vid:transcodedVideos`, so we would still have the problem with this check, right?

Reply:
Maybe you are right, I misunderstood the ticket. In that case I will check the mime-type of the `['vid:transcodedVideos']` propriety.